gcp/modules/gke-cluster/main.tf

```diff
@@ -34,7 +34,55 @@ variable "prevent_destroy_cluster" {
 }
 
 variable "enable_binary_authorization" {
-  default = false
+  default = true
+}
```

Comment:
I would only expose variables that are expected to be overwritten, not everything -> more readable, simpler. Can be always added later.